core/eval.go

```diff
@@ -87,6 +92,20 @@ func evalSET(args []string) []byte {
 				return Encode(errors.New("ERR value is not an integer or out of range"), false)
 			}
 			exDurationMs = exDurationSec * 1000
+		case "PX", "px":
+			if exDurationMs != -1 {
+				return Encode(errors.New("ERR syntax error"), false)
+			}
+			i++
+			if i == len(args) {
+				return Encode(errors.New("ERR syntax error"), false)
+			}
+
+			pxDurationMs, err := strconv.ParseInt(args[i], 10, 64)
```

Comment:
1. Cool
2. Since we are using `:=`, the VS code linter is giving error for `exDurationMs declared and not used`, since it's inside the `case PX` which might create  a temp local variable.